### PR TITLE
Pay Now fixes

### PR DESCRIPTION
**Pull request by jacksolirius**
### Change description ###
Fixed some of the issues in the 'SubmitAndPay' section for the C100 journey (PrL)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/gatling/resources/bodies/prl/c100/PRLSubmitAndPayDeclaration.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      "agree"
+    ]
+  },
+  "event": {
+    "id": "submitAndPay",
+    "summary": "",
+    "description": ""
+  },
+  "event_data": {
+    "applicantSolicitorEmailAddress": "perf-prl-aALof-user0@mailinator.com",
```
Comment (jonsoloway-hmcts) [inline src/gatling/resources/bodies/prl/c100/PRLSubmitAndPayDeclaration.json:22]: This should be replaced with ${user}

File: src/gatling/simulations/scenarios/Solicitor_PRL_C100.scala [hunk 2]
```diff
@@ -1291,17 +1292,20 @@ object Solicitor_PRL_C100 {
         .get("/data/internal/cases/${caseId}")
         .headers(Headers.commonHeader)
         .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-case-view.v2+json")
-        .header("x-xsrf-token", "${XSRFToken}"))
+        .header("x-xsrf-token", "${XSRFToken}")
+        .check(jsonPath("$.state.id").is("AWAITING_SUBMISSION_TO_HMCTS")))
```
Comment (jonsoloway-hmcts) [inline src/gatling/simulations/scenarios/Solicitor_PRL_C100.scala:1296]: You'll also need an additional check when you view the case to check that the latest event was added to the case successfully. e.g. .check(jsonPath("$.events........))

File: src/gatling/simulations/scenarios/Solicitor_PRL_C100.scala [hunk 3]
```diff
@@ -81,7 +81,7 @@ object Solicitor_PRL_C100 {
 
       .exec(Common.userDetails)
 
-      .exec(getCookieValue(CookieKey("XSRF-TOKEN").withDomain(BaseURL.replace("https://", "")).saveAs("XSRFToken")))
+  //    .exec(getCookieValue(CookieKey("XSRF-TOKEN").withDomain(BaseURL.replace("https://", "")).saveAs("XSRFToken")))
```
Comment (jonsoloway-hmcts) [inline src/gatling/simulations/scenarios/Solicitor_PRL_C100.scala:84]: This shouldn't be commented out

File: src/gatling/simulations/simulations/XUI_Simulation.scala [hunk 4]
```diff
@@ -62,7 +62,7 @@ class XUI_Simulation extends Simulation {
 	val caseworkerTargetPerHour:Double = 900
 
 	//This determines the percentage split of PRL journeys, by C100 or FL401
-	val prlC100Percentage = 66 //Percentage of C100s (the rest will be FL401s) - should be 66 for the 2:1 ratio
+	val prlC100Percentage = 100 //Percentage of C100s (the rest will be FL401s) - should be 66 for the 2:1 ratio
```
Comment (jonsoloway-hmcts) [inline src/gatling/simulations/simulations/XUI_Simulation.scala:65]: You need to undo these changes before you raise the PR, otherwise they'll be merged with master

File: src/gatling/simulations/simulations/XUI_Simulation.scala [hunk 5]
```diff
@@ -381,15 +381,15 @@ class XUI_Simulation extends Simulation {
...
-		ImmigrationAndAsylumSolicitorScenario.inject(simulationProfile(testType, iacTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
-		FamilyPublicLawSolicitorScenario.inject(simulationProfile(testType, fplTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
-		DivorceSolicitorScenario.inject(simulationProfile(testType, divorceTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
-		NoFaultDivorceSolicitorSoleScenario.inject(simulationProfile(testType, nfdSoleTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
-		NoFaultDivorceSolicitorJointScenario.inject(simulationProfile(testType, nfdJointTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
-		FinancialRemedySolicitorScenario.inject(simulationProfile(testType, frTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
-		CaseworkerScenario.inject(simulationProfile(testType, caseworkerTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption)
+		PRLSolicitorScenario.inject(simulationProfile(testType, prlTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption)
+//		ProbateSolicitorScenario.inject(simulationProfile(testType, probateTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
```
Comment (jonsoloway-hmcts) [inline src/gatling/simulations/simulations/XUI_Simulation.scala:385]: Same with this. You can't raise a PR with parts of the simulation commented out. Remember once this is merged with master, it'll impact the nightly pipeline.

**Review by jonsoloway-hmcts — COMMENTED** [review]
Please see comments and update the PR. Thanks.

File: src/gatling/simulations/scenarios/Solicitor_PRL_C100.scala (hunk 2 above)
Comment (jacksolirius) [reply src/gatling/simulations/scenarios/Solicitor_PRL_C100.scala:1297]: Done

File: src/gatling/simulations/simulations/XUI_Simulation.scala (hunk 5 above)
Comment (jacksolirius) [reply src/gatling/simulations/simulations/XUI_Simulation.scala]: Done

**Review by jonsoloway-hmcts — APPROVED** [review]
Looks good 👍
